docs/how-to/code/use-flask-extension/example/rockcraft.yaml

```diff
@@ -3,7 +3,11 @@ summary: A Flask application
 description: A rock packing a Flask application via the flask extension
 version: "0.1"
 base: bare
+build-base: ubuntu@22.04
```

Comment:
@weiiwang01 @gregory-schiano heads-up that we have a problem here: the context is in the [commit message](https://github.com/canonical/rockcraft/pull/521/commits/534bc41f4550ad628600b0bc4d51034b498dcdad). We should talk soon but after this is merged whatever projects are using the flask extension without a build-base (and a bare base) and/or no explicit platforms are going to break.

Reply:
cc @jdkandersson

Reply:
@tigarmo thanks for the note, I think it is fine. I propose we just get the extension to raise an error if the build-base/ platforms is missing. Any thoughts @gregory-schiano?

Reply:
@jdkandersson I see that you've already put up a PR for this, which is fine, but it's possibly redundant because a project that lacks build-base or platforms is going to fail validation before getting to the extensions now (this was the motivation for this original change that I had to do here). Still, no harm in doing two checks I suppose